client-lib/java/src/main/java/com/abcxyz/jvs/JvsConfiguration.java

```diff
@@ -35,7 +35,7 @@ public class JvsConfiguration {
   private String version = EXPECTED_VERSION;
 
   @JsonProperty("endpoint")
```

Comment:
It would be better if we update this config key as well to be `jwks_endpoint`

Reply:
Done.